commands/argument.go

```diff
@@ -37,6 +38,16 @@ func FileArg(name string, required, variadic bool, description string) Argument
 	}
 }
 
+func StreamArg(name string, required, variadic bool, description string) Argument {
```

Comment:
Yep totally! (btw this is being merged in to feat/pipe not master if you didn't see that, just wanted a easy way to get this to @whyrusleeping).